nncf/torch/quantization/default_quantization.py

```diff
@@ -13,21 +13,27 @@
 from typing import Dict, List
 
 from nncf.common.quantization.quantizer_propagation.structs import QuantizationTrait
+from nncf.common.graph.operator_metatypes import UnknownMetatype
 from nncf.torch.graph import operator_metatypes
 from nncf.torch.graph.operator_metatypes import PTOperatorMetatype
-from nncf.common.graph.operator_metatypes import UnknownMetatype
+from nncf.torch.graph.operator_metatypes import OPERATORS_WITH_WEIGHTS_METATYPES
 
 # If there are no some metatypes it means that they are considered as QuantizationTrait.QuantizationAgnostic
 
 DEFAULT_PT_QUANT_TRAIT_TO_OP_DICT = {
     QuantizationTrait.INPUTS_QUANTIZABLE: [
         operator_metatypes.PTConv2dMetatype,
+        operator_metatypes.PTModuleConv2dMetatype,
         operator_metatypes.PTConv3dMetatype,
+        operator_metatypes.PTModuleConv3dMetatype,
         operator_metatypes.PTConvTranspose2dMetatype,
+        operator_metatypes.PTModuleConvTranspose2dMetatype,
```

Comment:
Any reason why you decided to introduce module metatypes? make it more explicit and aligned with ordinary metatypes?

Reply:
The main issue is that to find out which metatypes have module subtypes and which haven't you should know how the mapping between NNCFModules and metatypes. Yes, in case someone decide to introduce new NNCFLayer one should implement module metatype as well, but explicit typing helps to identify is metatype has module subtype or not